### ISSUE TITLE
long_body - tests fail

### ISSUE THREAD

**Issue by pale-emperor** [tempesta-tech/tempesta#1659]
Long body have 4 tests which leads to FAIL condition:

```
long_body.test_long_request.RequestTest1M
long_body.test_long_request.RequestTest1k
```

[12199.646174] [tempesta fw] Warning: Parser error: state=RGen_BodyInit input(-8)=0xa('0.0.1
[12199.646175] [tempesta fw] Warning: failed to parse request: 127.0.0.1

Tests:
```
long_body.test_long_response.ResponseTest1k
long_body.test_long_response.ResponseTest1M

```
Also was disabled by another issue

I think we have a problems with tempesta-fw parser which leads to test fails

**Comment by krizhanovsky** [comment]
`Objects remaining` issue is gone in https://github.com/tempesta-tech/tempesta/pull/1643, but the parser errors are still with us.

**Comment by const-t** [comment]
Current tests completely reworked in https://github.com/tempesta-tech/tempesta-test/pull/304. Logging issue was fixed in PR #1418
